tests/TestCase.php

```diff
@@ -11,6 +11,7 @@ protected function getEnvironmentSetUp($app)
         $app['config']->set('blueprint.models_namespace', '');
         $app['config']->set('blueprint.app_path', 'app');
         $app['config']->set('blueprint.generate_phpdocs', false);
+        $app['config']->set('blueprint.use_constraints', true);
```

Comment:
This should default to `false` like the actual config and turned on in whatever test case you are verifying behavior of foreign key constraints.

Reply:
I'm currently busy with updating the tests at this point, changes will deployed in a little while!